docs/new-ansible-and-freezes.md

```diff
@@ -0,0 +1,146 @@
+# New Ansible Releases and Freezes
+
+## Preamble
+
+Releasing new Ansible major versions and frozen releases requires some special
+handling.
+For information about the general release process,
+see [Ansible Release Process](release-process.md).
+
+## Setting up for a new major release
+
+<!-- TODO: Write a script, playbook, or antsibull-build subcommand to automate this -->
+
+After the release of `X-1.0.0`, it is necessary to create the directory
+structure for Ansible `X`.
+
+1. Determine the current major version and previous major version
```

Comment:
I think current would mean X-1. Right now the current major version is 8, and will change to 9 on Tuesday when 9.0.0 is released. But we want to create the 10 directory, so we should say 'next major version' IMO.
```suggestion
1. Determine the next major version and current major version
```